invectio/lib.py

```diff
@@ -252,7 +254,7 @@ def _maybe_add(node: ast.AST):
                         f"Failed to parse assign statement in {self.file_name}"
                     )
 
-    def visit_AnnAssign(self, ann_assign: ast.AugAssign) -> None:
```

Comment:
This looks like a logic change. I think it should be kept named `visit_AnnAssign` to visit AnnAssign.

Reply:
thanks, fixed https://github.com/thoth-station/invectio/pull/93/commits/0a2d92553caf937cd2bb422c688c4714b3cfb074